### PR TITLE
[ector] Change symbols for ector

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: src/lib/ector/software/Ector_Software.h [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
+#  define ECTOR_API __declspec(dllimport)
 # endif
 #else
 # ifdef __GNUC__
 #  if __GNUC__ >= 4
-#   define EAPI __attribute__ ((visibility("default")))
+#   define ECTOR_API __attribute__ ((visibility("default")))
 #  else
 #   define EAPI
 #  endif
```
Comment (joaoantoniocardoso) [inline src/lib/ector/software/Ector_Software.h:26]: Should it use `EFL_BUILD` as a guard? If not, we could remove this part and include `ector_api.h`

Comment (Coquinho) [reply src/lib/ector/software/Ector_Software.h:26]: I think so too, but just to be sure will wait for another one to see before changing it.

File: src/lib/ector/gl/Ector_GL.h [hunk 2]
```diff
@@ -10,17 +10,17 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
 #  ifdef DLL_EXPORT
-#   define EAPI __declspec(dllexport)
+#   define ECTOR_API __declspec(dllexport)
```
Comment (Coquinho) [inline src/lib/ector/gl/Ector_GL.h:13]: Same as [here](https://github.com/expertisesolutions/efl/pull/179/files#r444524758)

**Comment by Coquinho** [comment]
> Maybe:
> 
> * add `-DECTOR_STATIC` to `src/lib/ector/software/meson.build:46`

Don't know about this, I will wait for some more comments.

> * add `-DECTOR_BUILD` to `src/lib/ector/meson.build:94`

Done

**Comment by Coquinho** [comment]
Hey @walac, there were changes after your approve, could you take a look again?

**Comment by walac** [comment]
> Hey @walac, there were changes after your approve, could you take a look again?

LGTM

**Pull request merged**